mlrun/runtimes/local.py

```diff
@@ -473,6 +473,13 @@ def get_func_arg(handler, runobj: RunObject, context: MLClientCtx, is_nuclio=Fal
     kwargs = {}
     args = inspect.signature(handler).parameters
 
+    def _get_input_value(input_key: str):
+        input_obj = context.get_input(input_key, inputs[input_key])
+        if type(args[input_key].default) is str or args[input_key].annotation == str:
+            return input_obj.local()
+        else:
+            return context.get_input(input_key, inputs[input_key])
```

Comment:
I believe it's the same
```suggestion
       return input_obj
```